### PR TITLE
feat(appeals): add notification banner entry point for appellant and lpa final comments

### PR DESCRIPTION
**Pull request by james-hill-kc**
## Describe your changes

add notification banner entry point for appellant and lpa final comments

## Issue ticket number and link

A2-1727

## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [ ] I have performed a self-review of my own code
- [ ] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [ ] I have provided details on how I have tested my code
- [ ] I have referenced the ticket number above
- [ ] I have provided a description of the ticket
- [ ] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: appeals/api/src/server/openapi.json [hunk 1]
```diff
@@ -3380,6 +3380,63 @@
 				}
 			}
 		},
+		"/appeals/{appealId}/reps/count": {
```
Comment (Si3rr4wow) [inline appeals/api/src/server/openapi.json:3383]: Thanks for filling this in!

File: appeals/web/src/server/appeals/appeal-details/accordions/index.js [hunk 2]
```diff
@@ -10,13 +10,17 @@ import { FEATURE_FLAG_NAMES } from '@pins/appeals/constants/common.js';
...
  * @returns {PageComponent}
  */
 export function generateAccordionItems(
 	appealDetails,
 	mappedData,
 	session,
-	ipCommentsAwaitingReview
+	ipCommentsAwaitingReview,
+	appellantFinalCommentsAwaitingReview,
+	lpaFinalCommentsAwaitingReview
```
Comment (Si3rr4wow) [inline appeals/web/src/server/appeals/appeal-details/accordions/index.js:23]: Maybe worth making an object for these? We're gunna have to pass them through for appellant and lpa statements to, and whatever else comes down the line

**Review by Si3rr4wow — DISMISSED** [review]

File: appeals/web/src/server/appeals/appeal-details/accordions/index.js (hunk 2 above)
Comment (james-hill-kc) [reply appeals/web/src/server/appeals/appeal-details/accordions/index.js:23]: Good idea, thanks! I'll make this update, but I also opened a tech debt ticket to eventually move the notification banner mapping code out of the accordion mapper: https://pins-ds.atlassian.net/browse/A2-1903

Comment (james-hill-kc) [reply appeals/web/src/server/appeals/appeal-details/accordions/index.js]: I've updated this to an object with the new type `RepresentationTypesAwaitingReview` now

**Review by Si3rr4wow — APPROVED** [review]

**Pull request merged**